### DISCUSSION TITLE
appearance-none does not remove basic arrow from select - Tailwind CSS : v3.2.2 - Standalone CLI

### DISCUSSION THREAD

**Discussion by ElliotPoinsignon** [tailwindlabs/tailwindcss#11602] [Help] [answered]
**What version of Tailwind CSS are you using?**

Tailwind CSS : v3.2.2 - Standalone CLI

**What build tool (or framework if it abstracts the build tool) are you using?**

Standalone CLI build using the watcher
`./tailwindcss -i input.css -o output.css --watch
`

**What version of Node.js are you using?**

N/A

**What browser are you using?**

Browser : Tested on Firefox and Chrome (same result on both)

**What operating system are you using?**

Browser on Windows
App on WSL2 Linux Ubuntu

**Reproduction URL**

Reproduction: https://play.tailwindcss.com/u66vWQGziq?size=540x720
The bug does not occure on tailwind playground but the setup and code are exactly the same as my app.

**Describe your issue**

Using `appearance-none` in the `select ` class  does not remove the base arrow of the select. Other strange phenomena : using `bg-none` remove the arrow with or without `appearance-none`. Since the same code works fine in the playground this might be an issue with the standalone CLI ?

Result on my app: 
With `appearance-none` and `bg-white` :
![image](https://github.com/tailwindlabs/tailwindcss/assets/64977087/b1d91653-7647-4bf5-92ee-5687d41fa0fd)
Without `appearance-none` and with `bg-none` :
![image](https://github.com/tailwindlabs/tailwindcss/assets/64977087/bcbf3214-4e25-4ed0-81a1-b8ab9ad1c403)

input.css
```
@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@700&display=swap');
@tailwind base;
@tailwind components;
@tailwind utilities;
```

**Comment by adamwathan** [comment, marked as answer]
Hey! Sounds like you have installed the `@tailwindcss/forms` plugin based on what you said about `bg-none`. If so, the chevron is added with a background image by that plugin which is why adding `bg-none` removes it.

**Comment by ElliotPoinsignon** [comment]
Hi, thanks for your quick answer, it was indeed the `@tailwindcss/forms` that caused this issue !

**Reply by adityamandal-Developer** [reply]
what did you do instead of @tailwindcss/forms removing it or anything else? i also want to remove that default arraw

**Reply by adityamandal-Developer** [reply]
okay i understand now but if i remove require("@tailwindcss/forms"), other things where its required will not work correctly right?

**Comment by Lysak** [comment]
I fixed this issue by adding `!` to `bg-none`, like `appearance-none !bg-none`.
